Sources_v3/StreamChatUI/ChatChannel/Reactions/ChatMessageReactionVC.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageReactionVC<ExtraData: ExtraDataTypes>: ViewController, UIConfigProvider {
+    public var messageController: _ChatMessageController<ExtraData>!
+
+    // MARK: - Subviews
+
+    public private(set) lazy var reactionsBubble = uiConfig
+        .messageList
+        .messageReactions
+        .reactionsBubbleView
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    // MARK: - Life Cycle
+
+    override open func setUp() {
+        super.setUp()
+
+        messageController.setDelegate(self)
+    }
+
+    override open func setUpLayout() {
+        view.embed(reactionsBubble)
+    }
+
+    override open func updateContent() {
+        reactionsBubble.content = messageController.message.flatMap { message in
```

Comment:
Why `flatMap` and not just `map`? I don't see any optionality in closure returning value

Reply:
**ChatMessageController** has `message?` that why the `flatMap` is here.